<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>200</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>320</width>
    <height>200</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string notr="true">ArmoryGS Updater</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/images/icon.ico</normaloff>:/images/icon.ico</iconset>
  </property>
  <property name="locale">
   <locale language="Russian" country="Russia"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="browseLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Variable</family>
      <pointsize>10</pointsize>
      <italic>true</italic>
      <bold>false</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>Путь до папки с установленным аддоном</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="browseLineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="browseButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>50</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Обзор</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Выключить обновление рейтингов</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="autoStartupCheckbox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Запускать при начальной загрузке Windows</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
